Client/src/Components/Inputs/Field/index.css

```diff
@@ -1,6 +1,15 @@
 .field {
 	min-width: 250px;
 }
+
+.field-infrastructure-alert {
+	max-width: 100px;
+}
+
+.field-infrastructure-alert .MuiInputBase-root:has(input) {
+	height: 34px;
+}
```

Comment:
I'm wary of hardcoding width and height into a component, I would prefer these be optional props with default values :thinking:

Reply:
The class name is a prop itself, I will see if I can define  the height value as css variable if that makes more sense

Reply:
It is better if you set it as min-height. Fixed heights are prone to overflow

Reply:
> It is better if you set it as min-height. Fixed heights are prone to overflow

To be consistent with normal input cases, I have used variable to set the height now

Reply:
Sorry, I don't understand, Jennifer. Why using height instead of min-height?
Using a fixed height can lead to overflow, like in this example
https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Overflowing_content#css_tries_to_avoid_data_loss